pygments/lexers/configs.py

```diff
@@ -70,6 +66,71 @@ def analyse_text(text):
         return text[0] == '[' and text[npos-1] == ']'
 
 
+
+class DesktopLexer(RegexLexer):
+    """
+    Lexer for .desktop files.
+    """
+
+    name = 'Desktop'
+    url = "https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html"
+    aliases = ['desktop']
+    filenames = ['*.desktop']
+
+    tokens = {
+        'root': [
+            (r'^[ \t]*\n', Whitespace),
+            (r'^(#.*)(\n)', bygroups(Comment.Single, Whitespace)),
+            (r'(\[[^\]\n]+\])(\n)', bygroups(Keyword, Whitespace)),
+            (r'([-A-Za-z0-9]+)([ \t]*)(=)([ \t]*)([^\n]*)([ \t\n]*\n)',
```

Comment:
I'd also like to see special support for localized entries of the form
```desktop
Name[de]=Hallo
```

Reply:
Yes, good idea. I'm not sure which token type would be suitable for this.

Reply:
Abusing the `Name.Namespace` for this for now.